### PR TITLE
Only use sha 512

### PR DESCRIPTION
**Pull request by mnm678**
I removed mentions of sha256 per #67 and redid the metadata overhead calculations.

Please double check my math on the metadata calculations.

### CODE REVIEW

File: pep-0458.txt [hunk 1]
```diff
@@ -307,9 +307,9 @@ kinds of metadata RECOMMENDED for PyPI.
 
 __ https://github.com/theupdateframework/tuf/blob/v0.11.1/docs/METADATA.md
 
-In addition, all target files SHOULD be available on disk at least three times.
+In addition, all target files SHOULD be available on disk at least two times.
 Once under their original filename, to provide backwards compatibility, and
-twice with their SHA-256 and SHA-512 hash respectively included in their
+twice with their SHA-512 hash included in their
```
Comment (joshuagl) [inline pep-0458.txt]:
```suggestion
once with their SHA-512 hash included in their
```

**Review by joshuagl — CHANGES_REQUESTED** [review]
Minor nit, otherwise looks good.

File: pep-0458.txt [hunk 2]
```diff
@@ -560,41 +557,39 @@ Table 1: A list of constants used to calculate metadata overhead.
...
-| V6   | Avg size of target paths per bin                                                   | V3*C6                        | 35,584    |
+| V5   | Avg size of target paths per bin                                                   | V3*C5                        | 35,584    |
 +------+------------------------------------------------------------------------------------+------------------------------+-----------+
-| V7   | Avg size of lengths per bin                                                        | V3*C7                        | 973       |
+| V6   | Avg size of lengths per bin                                                        | V3*C6                        | 973       |
 +------+------------------------------------------------------------------------------------+------------------------------+-----------+
-| V8   | Avg size of bin-n metadata (bytes)                                                 | V4+V5+V6+V7                  | 63,245    |
+| V7   | Avg size of bin-n metadata (bytes)                                                 | V4+V5+V76                    | 54,349    |
```
Comment (lukpueh) [inline pep-0458.txt]:
```suggestion
| V7   | Avg size of bin-n metadata (bytes)                                                 | V4+V5+V6                    | 54,349    |
```

**Review by lukpueh — CHANGES_REQUESTED** [review]
I cross-checked the numbers with the  [updated spreadsheet](https://docs.google.com/a/nyu.edu/spreadsheets/d/11_XkeHrf4GdhMYVqpYWsug6JNz5ZK6HvvmDZX0__K2I/edit?disco=AAAAD9TmS4), and they add up. There's a typo in one formula (see inline suggestions). Other than that LGTM! :)

**Review by lukpueh — APPROVED** [review]

**Pull request merged**